### PR TITLE
Refresh model

### PR DESCRIPTION
**Pull request by beto-devnet**

### CODE REVIEW

File: DataImport.Web/appsettings.json [hunk 1]
```diff
@@ -33,10 +33,10 @@
...
     }
   },
-  "ConnectionStrings": {
-    "defaultConnection": "Data Source=(local);Initial Catalog=EdFi_DataImport;Trusted_Connection=True",
-    "storageConnection": "UseDevelopmentStorage=true"
-  },
+    "ConnectionStrings": {
+        "defaultConnection": "Data Source=(local);Initial Catalog=EdFi_DataImport;Trusted_Connection=True;",
+        "storageConnection": "UseDevelopmentStorage=true"
+    },
```
Comment (nsd-sergio) [inline DataImport.Web/appsettings.json]: @beto-devnet  remove empty spaces.

File: DataImport.Web/Features/ApiServers/AddEditApiServer.cshtml [hunk 2]
```diff
@@ -92,6 +92,7 @@ See the LICENSE and NOTICES files in the project root for more information.
                         }
                     },
                     error: function(data) {
+                        console.log("error");
```
Comment (nsd-sergio) [inline DataImport.Web/Features/ApiServers/AddEditApiServer.cshtml]: @beto-devnet remove console.log

File: DataImport.Web/Features/ApiServers/RefreshDataModel.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+                var dataMaps = await (from dm in _database.DataMaps
+                                      join res in _database.Resources on dm.ApiVersionId equals res.ApiVersionId
+                                      where dm.ApiVersion == apiServer.ApiVersion && dm.ResourcePath.Trim() == res.Path.Trim()
+                                      select dm
+                                     ).ToListAsync(cancellationToken);
+
+                var resourcesPath = dataMaps.Select(map => map.ResourcePath).Distinct().ToList();
+
+                var metadataFromRresources = await _database
```
Comment (nsd-sergio) [inline DataImport.Web/Features/ApiServers/RefreshDataModel.cs]: fix spelling

File: DataImport.Web/Features/ApiServers/RefreshDataModel.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+                _database.DataMaps.UpdateRange(dataMaps);
+                await _database.SaveChangesAsync(cancellationToken);
+
+                return new Response
+                {
+                    ApiServerId = apiServer.Id,
+                    Message = "Data model was updated"
+                };
+
```
Comment (nsd-sergio) [inline DataImport.Web/Features/ApiServers/RefreshDataModel.cs]: remove empty space

File: DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            {
+                ApiServerId = apiServerId
+            });
+
+            refreshDataModelResponse.Message.ShouldBe("Data model was updated");
+            refreshDataModelResponse.ApiServerId.ShouldBe(apiServerId);
+        }
+
+        [TestCase(0)]
+        public void ShouldThrowAnExceptionWhenApiServerNotExist(int ApiServerId)
```
Comment (nsd-sergio) [inline DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs]: should be apiServerId

File: DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            var refreshDataModelResponse = await Send(new RefreshDataModel.Command
+            {
+                ApiServerId = apiServerId
+            });
+
+            refreshDataModelResponse.Message.ShouldBe("Data model was updated");
+            refreshDataModelResponse.ApiServerId.ShouldBe(apiServerId);
+        }
+
+        [TestCase(0)]
```
Comment (nsd-sergio) [inline DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs]: why TestCase[0] and not just Test?

**Comment by nsd-sergio** [comment]
Hi @beto-devnet

**Review by nsd-sergio — CHANGES_REQUESTED** [review]
Could you please review this changes

thanks

File: DataImport.Web/appsettings.json (hunk 1 above)
Comment (beto-devnet) [reply DataImport.Web/appsettings.json]: done

File: DataImport.Web/Features/ApiServers/RefreshDataModel.cs (hunk 3 above)
Comment (beto-devnet) [reply DataImport.Web/Features/ApiServers/RefreshDataModel.cs]: done

File: DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs (hunk 5 above)
Comment (beto-devnet) [reply DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs]: done

File: DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs (hunk 6 above)
Comment (beto-devnet) [reply DataImport.Web.Tests/Features/ApiServers/RefreshDataModelTests.cs]: done

**Pull request closed without merging**